lib/pure/strutils.nim

```diff
@@ -1108,6 +1394,17 @@ proc replace*(s: string, sub, by: char): string {.noSideEffect,
     else: result[i] = s[i]
     inc(i)
 
+proc expandTabs*(s: string, tabSize: int = 8): string {.noSideEffect,
```

Comment:
Thank you :) I didn't know that's how it's supposed to work. I will replace the current implementation with this.